notebooks/Lab8.ipynb

```diff
@@ -368,15 +388,15 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false,
+    "collapsed": true,
     "deletable": true,
     "editable": true
    },
    "outputs": [],
    "source": [
     "# load in data\n",
-    "neoantigens = Table.read_table('https://raw.githubusercontent.com/data-8/mcb-88-connector/gh-pages/data/lab8/TableS4.Neoantigen_list.csv').to_df()\n",
-    "\n",
+    "neoantigens = Table.read_table('https://raw.githubusercontent.com/data-8/mcb-88-connector/gh-pages/data/lab8/TableS4.Neoantigen_list.csv')\n",
+    "neoantigens = neoantigens.to_df() #Converting to Pandas dataframe\n",
```

Comment:
Why are we using a dataframe here and not in part 1? That may be confusing to students.